schema/rdl_schema_0.1.json

```diff
@@ -953,6 +949,23 @@
 				"Probable Maximal Loss"
 			]
 		},
+		"model": {
+			"type": "object",
+			"description": "The source model that produced the dataset.",
+			"properties": {
+				"title": {
+					"title": "Source model title",
+					"description": "The name of the model that the dataset originated from.",
+					"type": "string"
+				},
+				"date": {
+					"title": "Source model release date",
+					"description": "The date this dataset was published.",
```

Comment:
Should this be the date that the source model (rather than dataset) was published?